source/style-guide/style/lists.txt

```diff
@@ -39,8 +39,11 @@ guidelines when introducing lists.
        period instead of a colon.
 
        .. note::
-          Avoid using fragments to introduce lists. Fragments can be
-          harder to understand than sentences.
+
+          Where possible, avoid using fragments to introduce lists. 
+          However, you may use stem sentences to avoid especially 
```

Comment:
```suggestion
          However, you may use stem sentences to avoid especially 
```
Would be nice to have an example of an appropriate stem sentence. Wasn't familiar with the term!